archive/scripts/demoScripts/demoDeBridge.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '../typechain'
 import { node_url } from '../../utils/network'
 import chalk from 'chalk'
-import { config } from '../../config/debridge.json'
+import { config } from '../../../config/debridge.json'
```

Comment:
why there is a change the path for `config/debridge.json` but not for `typechain `and `import { node_url } from '../../utils/network'` above? Shouldn't they follow the same path structure?"

Reply:
ok, this change made me thinking that they still need to be runnable. All clear!